<?xml version="1.0" ?>
<launch>
  
  <arg name="sim" default="true" />
  <arg unless="$(arg sim)" name="topic_depth_registered_points" value="/ros_kinfu/depth_registered/points" />
  <arg     if="$(arg sim)" name="topic_depth_registered_points" value="/camera/depth_registered/points_" />


  <arg name="topic_segmented_points" default="/capsicum/points" />
    
  <node name="capsicum_segmentation" pkg="capsicum_detection" type="capsicum_segmentation" output="screen">
  
  <param name="topic_depth_registered_points" value="$(arg topic_depth_registered_points)" />
  <param name="topic_segmented_points" value="$(arg topic_segmented_points)" />


  </node>

</launch>

